env-files/tensorflow/createEnvJSCTF.sh

```diff
@@ -69,28 +62,10 @@ if [ "$cont1" = true ] ; then
   fi
 fi
 
-# install TF 
```

Comment:
Why is this removed?

Reply:
Ok I see that you moved the rest of it to the other file. But why not also keep the Horovod setup?